analysers/trials/cpu.py

```diff
@@ -37,22 +39,26 @@ def getActiveCores(sc, cassandraKeyspace, srcTable, trialID, experimentID, conta
                 nOfCpus += 1
         return nOfCpus
     
-    else:
-        nOfActiveCores = sc.cassandraTable(cassandraKeyspace, srcTable) \
-                .select("cpu_percpu_usage") \
-                .where("trial_id=? AND experiment_id=? AND container_id=? AND host_id=?", trialID, experimentID, containerID, hostID) \
+    else: 
+        nOfCpus = getHostCores(sc, cassandraKeyspace, hostID)
+        return nOfCpus
+
+def getHostCores(sc, cassandraKeyspace, hostID):
```

Comment:
You have the same function also in `analysers/experiments/cpu.py`, so it should go in `commons`